README.md

```diff
@@ -114,6 +114,22 @@ set :redirect_options, {
 Valid redirect destination should either start with `http` or `https` scheme,
 or begin with leading slash `/`.
 
+### Upload only compressed versions
+
+You can configure capistrano-s3 to only upload gzipped assets (when they are present) and remove the `.gz` suffix. This feature can come in handy when there is no server-side component to decide when to serve compressed or uncompressed content depending on `Accept-Encoding` header.
```

Comment:
Suggestion ; make it clear that in our specific case Amazon S3 does not provide this :

> This feature comes in handy because Amazon S3 does not provide a way to decide when to serve compressed or uncompressed content depending on `Accept-Encoding` header.